static/js/panels.js

```diff
@@ -79,9 +86,7 @@ export function initialize() {
     } else if (unread_ui.should_display_bankruptcy_banner()) {
         open($("[data-process='bankruptcy']"));
     } else {
```

Comment:
I would write this as `else if check_profile_incomplete()` for consistency with the other checks?  And maybe just inline `show_profile_incomplete` to have the else/if chaining be simpler?

Reply:
I guess i don't really understand why this case, and not the others, is the one with the `hide()` call -- do you understand why that's the case?

Reply:
I find that without the hide call the message does not go away even after writing description.The user has to reload the page to make that go away.

Reply:
I feel a more cleaner implementation is to reuse the <code>open</code> function.Something like 
```js
else if (check_profile_incomplete()){
        open($("[data-process='profile-incomplete']"));
    }
this works the same as the earlier approach.
```

Reply:
I think in <code>initialize()<code> we can do away with hide call. Its required only in <code>server_events_dispatch</code>.

Reply:
Yeah, this is great; I did a further change to use `open()` here just for consistency with the other code paths.